appserver/extras/grpc/pom.xml

```diff
@@ -49,14 +49,14 @@
     <artifactId>grpc</artifactId>
     <name>Payara gRPC Support</name>
     <packaging>jar</packaging>
-    <version>0.0.1</version>
+    <version>0.0.2</version>
```

Comment:
Not overly fond of non-SNAPSHOT version, particularly since this is recompiled as a part of the usual build.

Reply:
I upgraded the version with the SNAPSHOT strategy